internal/report/report.go

```diff
@@ -834,10 +834,23 @@ func printTraces(w io.Writer, rpt *Report) error {
 
 	_, locations := graph.CreateNodes(prof, &graph.Options{})
 	for _, sample := range prof.Sample {
-		var stack graph.Nodes
+		type stk struct {
+			*graph.NodeInfo
+			inline bool
+		}
+		var stack []stk
 		for _, loc := range sample.Location {
-			id := loc.ID
-			stack = append(stack, locations[id]...)
+			nodes := locations[loc.ID]
+			for i, n := range nodes {
+				// In this and some other reports the inline flag won't be accurate if
```

Comment:
Please open an issue for this if there isn't one open already.

Reply:
Filed https://github.com/google/pprof/issues/511.